scripts/clean-old-namespace-from-cluster.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env sh
+# this scripts logins as kubeadmin and removes the namespaces that are more than oneday old.
+# this script is used in a kubernetes cronjob for PSI/IBM cloud openshift cluster.  
+
+# login as kubeadmin
+if [[ $CLUSTER_TYPE == "PSI" ]]; then
+    ￼ #PSI cluster login
+    ￼ oc login -u kubeadmin -p ${OCP4X_KUBEADMIN_PASSWORD} --insecure-skip-tls-verify ${OCP4X_API_URL}
+else
+    ￼ # Login to IBM Cloud using service account API Key
+    ￼ ibmcloud login --apikey $IBMC_OCP47_APIKEY -a cloud.ibm.com -r eu-de -g "Developer CI and QE"
+    ￼
+    ￼ # Login to cluster in IBM Cloud using cluster API key
+    ￼ oc login --token=$IBMC_OCLOGIN_APIKEY --server=$IBMC_OCP47_SERVER
+fi
+
+# PROJECT_AND_TIME var will contain namespace and date with time seperated with `|`
+# PROJECT_AND_TIME doesn't contain openshift/ibm/kube namespace
+# eg. cmd-push-test157kgb|2021-08-17T12:40:20Z
+PROJECT_AND_TIME=$(kubectl get projects -o jsonpath='{range .items[*]}{.metadata.name}{"|"}{.metadata.creationTimestamp} {"\n"}{end}' | grep -v '^openshift\|^kube\|^default\|^ibm\|^calico\|^tigera')
+
+for PROJECT in ${PROJECT_AND_TIME}; do
+    IFS='|'
+    read -ra ADDR <<<"$PROJECT"
```

Comment:
Thanks for adding `eg. cmd-push-test157kgb|2021-08-17T12:40:20Z` above. Can you add comments within this function that would help understand what the value of `ADDR` will be based on that example?

Similarly, for other variables below like `datetime`, `dtSec`, `taSec` as well? I'm asking this only because when someone other than you takes a look at this code a few weeks/months later, these comments would make it really simpler for them to understand what changes need to be made.

Reply:
understood will add more comments to the code.